docs/genesis-preparation.md

```diff
@@ -0,0 +1,366 @@
+certificate allows to delegate stake from `SK` to `DPK`, where `SK` is a stakeholder's
+secret key and `DPK` is a delegate's public key. Stakeholder's secret key was generated
+in step `2.1`, and delegate's public key was obtained in step `1.2`.
+
+Firts of all, you should add the key by this command:
+
+```
+$ stack exec -- cardano-auxx cmd --commands add-key PATH_TO_SK 
+```
+
+where `PATH_TO_SK` is a path to stakeholder's secret key generated in step `2.1`. After that
+you can generate delegation certificate. It can be produced by the command:
+
+```
+$ stack exec -- cardano-auxx cmd --commands delegate-heavy SKN DPK EI dump
+```
+
+where:
+
+1.  `SKN` - number of issuer's secret key,
+2.  `DPK` - delegate's public key,
+3.  `EI` - epoch's index (for Mainnet launching it is 0).
+
+Example of command:
+
+```
+$ stack exec -- cardano-auxx cmd --commands                                                     \
+    delegate-heavy                                                                              \
+    0                                                                                           \
+    bNhQZ5x9DGYeV4gLyk1Axa4yp4xzrzZBskXov4Vk7AkrogndKgpkiMLJrNqaCJQAJmyrzAbeq4WtqkYH7mBisM2e2w  \
+    0                                                                                           \
+    dump
+```
+
+Example of response:
+
+```
+pskDelegatePk: hecJCq/AX2Xn43dP7yqxK0wFQfFkNv3tM0cu/kG4xgzEvOO0coIh8epYeHdxCcoxmKIvmTJILqwJv1M+IsA4jA==
+pskOmega:
+  getEpochIndex: 0
+pskCert: 1d2d94b625afcfaa127974e1c6b49734cca1056c37a2228fd737e16611ca6bb78f1c254b400c5b0c9aeb3a2c9454d72eda8246aed0bd3be0cacb6f69a6475c0a
+pskIssuerPk: LGPdjVPmSRMTG0uK33kwr/ZCgo9zTWTAa8d1hjqGjIcenTQoblqtlVFdbLOkDdKaOnDOs0XLpq1mhyNr5Mki+Q==
+```
+
+Then delegation certificate and stakeholder's (issuer's) public key should be published in any
+public channel: GitHub Gist, website, etc. Later these delegation certificate and stakeholder's
+public key will be included in genesis block (see below).
+
+**Responsible**: Each of the parties' (IOHK/CGG/CF) DevOps teams.
+
+## 3. Finalize AVVM utxo
+
+### 3.1. IOHK/CGG/CF Wallet Keys
+
+The purpose of the wallet keys is an operations with money.
+
+To generate wallet keys use this command:
+
+```
+$ stack exec -- cardano-keygen generate-avvm-seeds -n 3 -o PATH_TO_OUTPUT_DIR
+```
+
+where `PATH_TO_OUTPUT_DIR` is the path to output directory where generated keys will be
+stored. Argument `-n` defines number of generated keys, in this case we will get 3 keys.
+
+After that output directory will contain these files:
+
+```
+key1.pk  key1.seed
+key2.pk  key2.seed
+key3.pk  key3.seed
+```
+
+Files with `.pk` suffix are AVVM public keys. Files with `.seed` suffix are corresponding AVVM seeds.
+
+AVVM seeds will be used to produce redemption certificates.
+
+**Responsible**: IOHK DevOps team.
+
+### 3.2. Blacklist utxo
+
+Some of `utxo`'s entries will be added in the AVVM blacklist. Example of such a blacklist can be found
```

Comment:
There is no more blacklist. Just a final utxo. Leave that utxo should be refined with last inclusions/exclusions of existing addresses